### ISSUE TITLE
pypi.streamsx.hbase: Provide composite class

### ISSUE THREAD

**Issue by anouri** [IBMStreams/pypi.streamsx.hbase#10]
uses streamsx feature for pypi.streamsx.hbase:
Support the concept of composite transformations.
IBMStreams/streamsx.topology#2322

**Comment by anouri** [comment]
The pypi.streamsx.hbase improved with 3 new classes:
- HBaseGet 
- HBasePut
- HBaseScan

3 new test have been added to check the new feature.
```
cd package
python3 -u -m unittest streamsx.hbase.tests.test_hbase.TestCompositeClass
```

**Comment by anouri** [comment]
corrected in version 1.50
https://github.com/IBMStreams/pypi.streamsx.hbase/releases/tag/1.5.0
